app/controllers/traces_controller.rb

```diff
@@ -1,6 +1,7 @@
 class TracesController < ApplicationController
   include UserMethods
   include PaginationMethods
+  include ActionView::Helpers::TagHelper
```

Comment:
Is this include still needed after the fix https://github.com/openstreetmap/openstreetmap-website/pull/3429#discussion_r2106266400?